src/js/game/LevelMVC/LevelView.js

```diff
@@ -1025,6 +1025,10 @@ module.exports = class LevelView {
       const group = block.shouldRenderOnGroundPlane() ? this.groundGroup : this.actionGroup;
       const offset = block.shouldRenderOnGroundPlane() ? -0.5 : 0;
       sprite = this.createBlock(group, position[0], position[1] + offset, blockType);
+      if (blockType.startsWith('redstoneWire') && blockType.endsWith('On')) {
+        // Please excuse the magic numbers. They're just to offset the sparkle correctly within the index.
+        sprite.addChild(this.createBlock(group, 0.35, 0.05, "redstoneSparkle"));
+      }
```

Comment:
Actually, I think we want to move this logic to `createBlock`?  So that however createBlock is called it will always create the sparkle animation for powered wires.